test/kg/accounts_test.exs

```diff
@@ -92,6 +92,16 @@ defmodule Kg.AccountsTest do
       assert is_nil(user.password)
     end
 
+    test "registers users with a first and last name" do
+      email = unique_user_email()
+      first_name = "Tim"
+      last_name = "Doyle"
+      {:ok, user} = Accounts.register_user(%{email: email, password: valid_user_password()})
```

Comment:
You defined `first_name` and `last_name` variables which is a good first start. But you need to add these into the parameters to register_user. You'll see that this test fails when running `mix test`